internal/common/compilation.bzl

```diff
@@ -292,6 +290,34 @@ def compile_ts(ctx,
   if not is_library:
     files += depset(tsickle_externs)
 
+  rerooted_es6_sources = []
+  es6_js_modules_roots = [] # todo(achew22): this is non recursive... recurse.
```

Comment:
Use this code as a base to make this recursive: https://github.com/bazelbuild/rules_closure/blob/master/closure/private/defs.bzl#L61